src/action/interactive.rs

```diff
@@ -272,7 +272,45 @@ impl UserPicked {
                 KeyCode::Char('j') => return Ok(Pick::Previous),
                 KeyCode::Char('q') | KeyCode::Esc => return Ok(Pick::Quit),
                 KeyCode::Char('d') => return Ok(Pick::SkipFile),
-                KeyCode::Char('e') => unimplemented!("Manual editing is a TODO"),
+                KeyCode::Char('e') => {
+                    let guard = ScopedRaw::new();
+                    stdout()
+                        .queue(cursor::MoveToNextLine(0))    
+                        .unwrap()
+                        .queue(terminal::Clear(terminal::ClearType::CurrentLine))
+                        .unwrap()
+                        .queue(Print("replacement: "))
+                        .unwrap();
+                    let _ = stdout().flush();
+                    drop(guard);
+
+                    let mut replacement = String::new();
+                    loop {
+                        let event = match crossterm::event::read()
+                            .map_err(|e| anyhow::anyhow!("Something unexpected happened on the CLI: {}", e))?
+                        {
+                            Event::Key(event) => event,
+                            sth => {
+                                trace!("read() something other than a key: {:?}", sth);
+                                break;
+                            }
+                        };
+            
+                        trace!("registered event: {:?}", &event);
+                        let KeyEvent { code, modifiers: _ } = event;
+            
+                        match code {
+                            KeyCode::Enter => break,
+                            KeyCode::Char(c) => {
+                                replacement.push(c);
```

Comment:
> so typos while fixing typos do not require another run of cargo-spellcheck.

Yes that's a good aim. But could you elaborate the ideas behind?
>adding arrow key, delete and insert mode too

Reply:
I tented to learn more about `adding arrow key, delete and insert mode`.

As I understand by adding arrow key you mean add a line right after suggestions marked by arrow.

```
   ...
   stout
» custom replacement
```

But I can't recognise what should be done with `delete and insert mode`.
Thanks.

Reply:
When you hit the `insert` physical key on your keyboard, the mode changes from inserting to overwriting, it might be overkill to impl, but some people have that in their workflow - not mandatory and could be done separately